app/utils/security.ts

```diff
@@ -7,19 +7,3 @@ export async function getCSRFFromCookie(url: string) {
     const cookies = await CookieManager.get(url, false);
     return cookies.MMCSRF?.value;
 }
-
-export const hashCode = (str: string): string => {
```

Comment:
I wonder if we should keep this and add a deprecated comment.

The reason to keep it as cause we can use it to delete the previous database files and cache folders.